repoclosure/yum.conf

```diff
@@ -37,19 +37,12 @@ name=Foreman Client nightly EL9
 baseurl=https://stagingyum.theforeman.org/client/nightly/el9/x86_64/
 
 [el8-baseos]
-name=BaseOS
-enabled=1
-mirrorlist=http://mirrorlist.centos.org/?release=8-stream&arch=x86_64&repo=BaseOS
-#baseurl=http://vault.centos.org/centos/8/BaseOS/$arch/os/
-failovermethod=priority
+name=AlmaLinux 8 - BaseOS
+mirrorlist=https://mirrors.almalinux.org/mirrorlist/8/baseos
 
 [el8-appstream]
-name=AppStream
-enabled=1
-mirrorlist=http://mirrorlist.centos.org/?release=8-stream&arch=x86_64&repo=AppStream
-#baseurl=http://vault.centos.org/centos/8/AppStream/$arch/os/
-failovermethod=priority
-module_hotfixes=1
```

Comment:
this broke repoclosure :cry: 

https://ci.theforeman.org/blue/organizations/jenkins/foreman-plugins-nightly-rpm-pipeline/detail/foreman-plugins-nightly-rpm-pipeline/1819/pipeline/39/

Reply:
I just noticed the same thing when I was doing the same for pulpcore.

Reply:
https://github.com/theforeman/foreman-packaging/pull/10886